### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by LethaboMot**
In this template I:

-all the commits are ahead
-correct my README
-corrected the stylelint indentation

### CODE REVIEW

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @LethaboMot, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that has failed a linter check. Kindly fix your linter errors and submit again 👍🏽  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,63 +1,64 @@
...
+        </div>
+
+        <div>
+
+
+
+            <header class="mid">
+                <h1 align=" center">I'M LETHABO</h1>
+                <h1 align=" center">GLAD TO SEE YOU!</h1>
+                <img src="images/Primary Text2.svg" alt="text" align="center">
```
Comment (Olamarx) [inline index.html:37]: - [ ] This image should be used in the styling. If an image is not part of the content (for aesthetic reasons) it should be used as a CSS background image.

File: index.html [hunk 2]
```diff
@@ -1,63 +1,64 @@
...
+                <img src="images/connect.svg" alt="text" align="center">
```
Comment (Olamarx) [inline index.html:38]: - [ ] This image should be used in the styling. If an image is not part of the content (for aesthetic reasons) it should be used as a CSS background image.

File: index.html [hunk 3]
```diff
@@ -1,63 +1,64 @@
...
+
+    <div class="app-wrap">
+        <div class="topnav">
+            <p style="color: white">space</p>
+            <p style="color: white">space</p>
+            <a href="button">
+                <img src="images/My Logo.svg">
+            </a>
+            <a href="#" class="Split">
+                <img src="images/Hamburger button.svg">
```
Comment (Olamarx) [inline index.html:26]: - [ ] This image should be used in the styling. If an image is not part of the content (for aesthetic reasons) it should be used as a CSS background image.

File: index.html [hunk 4]
```diff
@@ -1,63 +1,64 @@
...
+                <img src="images/connect.svg" alt="text" align="center">
+
+            </header>
+            <div class="socials">
+                <a href="#"> <img src="images/Twitter.svg"></a>
+                <a href="#"><img src="images/Linkedin.svg"></a>
+                <a href="#"><img src="images/Medium.svg"></a>
+                <a href="#"><img src="images/GitHub.svg"></a>
+                <a href="#"><img src="images/Angel list.svg"></a>
+            </div>
```
Comment (Olamarx) [inline index.html:47]: - [ ] Kindly replace the div with ul tag and wrap the a tags in li tag. This is for semantic reasons.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @LethaboMot, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linters passed ✔️ 
- Meaningful commit messages ✔️ 
- Good use of Git and Github flows ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -1,44 +1,102 @@
...
+header .toolbar a {
+  text-decoration: none;
+  color: #636fd4;
+}
+
+header {
+  margin: 0 auto;
+  height: 48px;
+  padding: 0 24px;
+}
```
Comment (PinkMoon25) [inline style.css:33]: [OPTIONAL] Currently your header section does not have any padding-top, and is sticking to the top like this :point_down: 
image
how it is supposed to be :point_down: 
image
you can fix this by setting the padding on your header like this :point_down: 
```
header {
  padding: 24px;
}
```

File: style.css [hunk 6]
```diff
@@ -1,44 +1,102 @@
...
+#Introduction {
+  width: 100%;
+  height: calc(100vh - 48px);
+  background-image: url("images/background.svg");
+  background-size: 100% 100%;
+  padding-top: 114px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
```
Comment (PinkMoon25) [inline style.css:44]: [OPTIONAL] Currently your intro section does not have any padding-left and padding-right and is sticking at both side like this :point_down: 
image
It should be like this :point_down: 
image
You can fix this by removing `width:100% `and adding a `padding: 0 22px; `

**Review by PinkMoon25 — APPROVED** [review]
Hi @LethaboMot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters passed :heavy_check_mark: 
Meaningful commit messages  :heavy_check_mark: 
Good use of Git and Github flows  :heavy_check_mark: 
used flexbox for the layout  :heavy_check_mark: 
follows figma design  :heavy_check_mark: 

## Optional suggestions
#### _**You have done a good job implementing the changes, I have added some Optional changes please take them into consideration in order to Improve our site and make it more similar to the figma template**_
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — COMMENTED** [review]
Hi @LethaboMot, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that not corresponding to the activity. You submit the project the activity name is `Portfolio: finish mobile version` and I think you should submit `Portfolio: setup and mobile first` 🙃

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Favourezeugwa — COMMENTED** [review]
Hi @LethaboMot , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌  ❌ 

You have submitted the wrong project. This pull request contains your `portfolio set-up` branch  and not the `finish mobile version`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**